docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       - ALLOWED_HOSTS=["*"]
       - CORS_ORIGIN_WHITELIST=('localhost:3000',)
       - GATHER_CASES_CRON_TIME_SECONDS=1440
+      - FACEBOOK_APP_ID=108193869861396
+      - FACEBOOK_APP_SECRET=6c7e431f5dda1d4075f027fe0b667ee8
```

Comment:
Is this to the "production" environment of the Facebook app? As far as I remember, you can have a DEV and a Production version.

I dont think we should store the ID and Secret of the production app anywhere in the source code, but the DEV app should be fine I think